modules/oadp-backing-up-pvs-csi.adoc

```diff
@@ -5,8 +5,6 @@
 :_content-type: PROCEDURE
 [id="oadp-backing-up-pvs-csi_{context}"]
 = Backing up persistent volumes with CSI snapshots
-include::_attributes/common-attributes.adoc[]
```

Comment:
I confirmed the attributes were called correctly in the assembly, then deleted these two lines from the module.